src/test/java/uk/gov/hmcts/sptribs/common/notification/NotificationHelperTest.java

```diff
@@ -293,6 +298,18 @@ void shouldAddHearingPostponedTemplateVars() {
         assertThat(templateVars.get(HEARING_TIME)).isEqualTo("11:00");
     }
 
+    @Test
+    void shouldAddHearingPostponedTemplateVarsWithNull() {
```

Comment:
A new test case needs to be added with invalid input (wrong format) for selected hearing date and time.